packages/richtext-lexical/src/exports/react/components/RichText/converter/converters/text.tsx

```diff
@@ -0,0 +1,38 @@
+import type { SerializedTextNode } from 'lexical'
+
+import escapeHTML from 'escape-html'
+import React from 'react'
+
+import type { JSXConverters } from '../types.js'
+
+import { NodeFormat } from '../../../../../../lexical/utils/nodeFormat.js'
+
+export const TextJSXConverter: JSXConverters<SerializedTextNode> = {
+  text: ({ node }) => {
```

Comment:
might be nice to make a pattern where `text` could optionally return undefined, and if it did, we could assume that we should run the default converter